vmck/backends/qemu.py

```diff
@@ -73,6 +73,17 @@ def task_group(job, options):
         '-device', 'virtio-9p-pci,fsdev=vmck,mount_tag=vmck',
     ]
 
+    if options['usbstick']:
+        usbsticks = Path(settings.QEMU_USBSTICKS).resolve()
+        image = (usbsticks / options['usbstick']).resolve()
+        assert usbsticks in image.parents
```

Comment:
This seems like a no-op if the `.resolve()` calls above don't throw. How about `image = (Path.QEMU_USBSTICKS / options['usbstick']).resolve(); assert image.exists()`?

Reply:
Its only purpose is to make sure the user doesn't escape outside the images folder. I'll add a note to the assert.